modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
-* Total Failed Inbound Requests grouped by endpoint
-* Failed Outbound Requests: Response time by HTTP endpoint (only HTTP endpoints are shown)
-* Total Failed Outbound Requests grouped by HTTP endpoint (only HTTP endpoints are shown)
-| *JVM* a|
-
-* Garbage collection count, Garbage collection time
-* Par new collection count, Par new collection time
-* Classes: Loaded, Loaded total, Unloaded
-* Heap: Committed, Used
-* Thread Count: Number of simultaneous requests
-* JVM Uptime: Uptime of the JVM
-* Par Eden graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
-* Par Survivor graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
-* Metaspace graphs (Used, Max, Init, Committed): Usage metrics for Java memory space
-* Code Cache graphs (Used, Max, Init, Committed): Usage metrics for the Code Cache pool
-* Compressed Class Space graphs (Used, Max, Init, Committed): Usage metrics for values of the metadata for compressed class pointers
-* Committed Virtual Memory: Amount of virtual memory guaranteed to be available to the running process
-| *Infrastructure* a|
-
-* Application Process CPU Load: Amount of CPU load used by the JVM
-* Application Process CPU Minutes: CPU time used by the process on which the JVM is running
-* Operating System Swap Space: Amount of memory the system is swapping
-* CPU Utilization %: Percentage of CPU used by each worker over time
-* Memory Utilization: Amount of memory used by each worker over time
-* Total System Processors: Number of system processors for workers available over time
-* Total System Memory: Amount of system memory available for workers over time. +
-If you have a Titanium subscription and the selected application is hybrid, this graph changes to *System Memory*.
-* System Memory: This graph displays only if you have a Titanium subscription and the selected application is hybrid. This graph displays the total free system memory per worker.
-* Thread Count - Server: Number of simultaneous requests for a worker over time
+* All Inbound Requests Grouped by Response Type
+* All Outbound Requests Grouped by Response Type
+* Failed Inbound Requests: Response Time by Endpoint
+* Total Failed Inbound Requests Grouped by Endpoint
+* Failed Outbound Requests: Response Time by HTTP Endpoint
+* Total Failed Outbound Requests Grouped by HTTP Endpoint
+| <<jvm-charts>> a|
+
+* Garbage Collection Count 
+* Garbage collection Time
+* Par new Collection Count
+* Par new collection Time
+* Classes (Loaded, Loaded Total, Unloaded)
+* Heap (Committed, Used)
+* Thread Count
+* JVM Uptime
+* Par Eden graphs (Used, Max, Init, and Committed)
+* Par Survivor graphs (Used, Max, Init, and Committed)
+* Metaspace graphs (Used, Max, Init, Committed)
+* Code Cache graphs (Used, Max, Init, Committed)
+* Compressed Class Space graphs (Used, Max, Init, Committed)
+* Committed Virtual Memory
+| <<infrastructure-charts>> a|
+
+* Application Process CPU Load
```

Comment:
@myuin-ms UX needs to fix this chart to: https://www.mulesoft.org/jira/browse/MONITORING-3893?filter=-2